adafruit_ds3231.py

```diff
@@ -96,14 +97,10 @@ class DS3231:
     def __init__(self, i2c):
         self.i2c_device = I2CDevice(i2c, 0x68)
 
-        # Try and verify this is the RTC we expect by checking the rate select
-        # control bits which are 1 on reset and shouldn't ever be changed.
-        buf = bytearray(2)
-        buf[0] = 0x0e
-        with self.i2c_device as i2c_device:
-            i2c_device.write_then_readinto(buf, buf, out_end=1, in_start=1)
-
-        if (buf[1] & 0b00011000) != 0b00011000:
+        # Try and verify this is the RTC we expect by checking change in secs
+        check = self.datetime_register.tm_sec
```

Comment:
could it be stopped?

Reply:
That's an interesting question.  It's possible, if the EOSC bit at 0x0E is set to 0 and the clock is on battery power only, then the clock would be stopped.  I think maybe go back to the idea of a parameter being given to the constructor...verify_running=true?  If true (default), the constructor verifies that the time is changing and so we have a running clock.  If false, then the user clearly wants to be ok just having an item on 0x68.

Reply:
good point. could be. hmmm. can be checked with OSF in status reg? but makes me question this general approach?

Reply:
delete it all, use just the addr

Reply:
so no "is it a DS3231" check? just "yep. something's at 0x68"?

Reply:
yep!

Reply:
done. address check covered by I2CDevice init.